src/routes/dispensary/add/+page.server.ts

```
@@ -21,15 +21,10 @@ export const actions = {
             city: formCity,
             region_id: formState,
             postal_code: formZip,
-            validation_status_id: formStatus,
-            created_by: formCreatedBy
+            validation_status_id: 2, //pending
+            created_by: "22222222-2222-2222-2222-222222222222", // placeholder user id
           },
         ]);
-
-      if (error) {
```

Don't ignore the error, throw it!
```
if (error) {
  throw error
}
```